app/models/Event.js

```diff
@@ -82,14 +80,21 @@ const eventSchema = new Schema({
     }
   },
   eventDate: {
-    type: Date,
+    type: String,
```

Comment:
This should be Date and in the test .js you can use momentjs to get current date in a specific format

Reply:
sir, I have already added the validation for the string , so now it will be saved but it will happen only when the validation of date formatting will be passed
```
  eventDate: {
    type: String,
    required: true,
    validate (eventDate) {
      if (!validator.isISO8601(eventDate)) {
        throw new Error('Event date is not formated!')
      }
      if (validator.isEmpty(eventDate)) {
        throw new Error('Event date is required!')
      }
    }
  }
```

Reply:
![image](https://user-images.githubusercontent.com/48439116/80481514-38c50580-8970-11ea-8901-ba7180fa321b.png)
here is the thing i was discussing before , date has been saved in Date format

Reply:
very well sir, then i will remove the validation from it because validation required string as an input

Reply:
sure,make a validation on controller that date entered should not be less then current date

Reply:
sir, i believe this issue should be take care at the frontend at the first case
because user should be stoped at the moment he/she is trying to enter the old date
otherwise , it will be a waste of time for the user that  they filled the whole form and later getting the errors
so i think , at the frontend , in the datepicker , date before the current date should be disabled

Reply:
got it